### DISCUSSION TITLE
TN3270 performance

### DISCUSSION THREAD

**Discussion by rduke26** [zowe/community#1401] [Q&A]
In regard to using the Zowe emulator as a production tool I was asked if it is "production ready".  In this regard is there any data regarding performance or scalabiltiy? For example how many concurrent users sessions can be supported (any architectural limitation?) and some understanding of memory and cp/ziip required?   Are there any limitations in function or securing the sessions?  Thanks in advance.

**Comment by adriankadev** [comment]
Hi @rduke26,
I'm also very interested to get more information about the performance with this solution and saw also you post in https://github.com/zowe/zlux/discussions/769 . If something is available like in the following repo this would be great https://github.com/plavjanik/zowe-zosmf-perftest-driver or if someone has some experience with the technical settings and amount of users.
Best regards
Adrian
